bodhi/__init__.py

```diff
@@ -61,8 +61,8 @@ def get_user(request):
 
 
 def groupfinder(userid, request):
-    user = request.user
-    if user:
+    if request.user:
+        user = User.get(request.user.name, request.db)
```

Comment:
Yup, I think we need to import it at a function level to avoid a circular dependency. :repeat: